head.html

```diff
@@ -195,6 +241,7 @@
         fetchTopics();
       case pageTypes[1]: // authors
         fetchSocialLinks();
+        fetchProducts();
```

Comment:
Isn't this the wrong place? I would have expected this to happen on the post pages, not the author pages...